webapp/channels/src/components/post/actions.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {DispatchFunc} from 'mattermost-redux/types/actions';
+import {ExtendedPost, removePost} from 'mattermost-redux/actions/posts';
+
+import {isThreadOpen} from 'selectors/views/threads';
+import {getGlobalItem} from 'selectors/storage';
+
+import {removeDraft} from 'actions/views/drafts';
+import {closeRightHandSide} from 'actions/views/rhs';
+
+import {StoragePrefixes} from 'utils/constants';
+
+import {GlobalState} from 'types/store';
+
+export function removePostCloseRHSDeleteDraft(post: ExtendedPost) {
```

Comment:
extracted and constructed on this function which also deletes the draft

Reply:
Yep, that makes sense to me. Having it in the index file is fine enough when there's not much going on in there, but there's already a lot happening in that file, so splitting it out makes sense to me